### ISSUE TITLE
Missing favicons

### ISSUE THREAD

**Issue by Moris299** [opencaching/opencaching-pl#1547]
There are no favicons for:

-Android homescreen
-Windows Metro
-macOS Safari (touchbar and Pinned tab)
-Opera Browser (Speed Dial and Bookmarks)

This favicons will make Openacaching shortcuts looks better on many browesers and OS.

**Comment by andrixnet** [comment]
```
  <link rel="shortcut icon" href="/images/oc_icon-ro.png">
  <link rel="apple-touch-icon" sizes="180x180" href="/images/icons/apple-touch-icon.png">
  <link rel="icon" type="image/png" sizes="32x32" href="/images/icons/favicon-32x32.png">
  <link rel="icon" type="image/png" sizes="16x16" href="/images/icons/favicon-16x16.png">
  <link rel="manifest" href="/images/icons/site.webmanifest">
  <link rel="mask-icon" href="/images/icons/safari-pinned-tab.svg" color="#5bbad5">
```

This modification does not take into account per-node favicons.
Only the first item is set per-node. All the rest use the same image for each node.

Also, the presence of the new tags seems to be preferred, also by desktop browsers, and thus almost completely cancel per-node favicon personalization.

**Comment by kojoty** [comment]
@Moris299, yeah, andrixnet is right - it seems we should allow nodes to customize such icons by settings...

@andrixnet - thanks - I forget about it!
